src/outputs/parthenon_hdf5.cpp

```diff
@@ -36,6 +36,10 @@
 #include "outputs/parthenon_hdf5.hpp"
 #include "utils/string_utils.hpp"
 
+#define GB            (1024 * 1024 * 1024)
+#define MB            (1024 * 1024)
+#define KB            (1024)
```

Comment:
Is there a reason to use macros here, instead of `constexpr int`?

In the latter case, perhaps we should move these into global `decs` so they're available everywhere?

Reply:
No reason. If you think they would be helpful generally, it would be good to make them available.

Reply:
I think I like putting this [here](https://github.com/lanl/parthenon/blob/develop/src/defs.hpp) but it's also fine to leave it where it is.

Reply:
updated and moved.